rackcity/views/report_views.py

```diff
@@ -60,34 +63,30 @@ def compute_rack_report(datacenter_id=None):
             owner_allocation[owner] = 0
         owner_allocation[owner] += height
 
-    rack_usage_data = {}
-    rack_usage_data["free_rackspace_percent"] = 1 - (
-        num_full_rack_slots / total_num_rack_slots
-    )
-
-    rack_usage_data["vendor_allocation"] = [
-        {
-            "vendor": vendor,
-            "allocation_percent": vendor_allocation[vendor] / num_full_rack_slots,
-        }
-        for vendor in vendor_allocation
-    ]
-
-    rack_usage_data["model_allocation"] = [
-        {
-            "vendor": model.vendor,
-            "model_number": model.model_number,
-            "allocation_percent": model_allocation[model] / num_full_rack_slots,
-        }
-        for model in model_allocation
-    ]
-
-    rack_usage_data["owner_allocation"] = [
-        {
-            "owner": owner,
-            "allocation_percent": owner_allocation[owner] / num_full_rack_slots,
-        }
-        for owner in owner_allocation
-    ]
+    rack_usage_data = {
+        "free_rackspace_percent": 1 - (num_full_rack_slots / total_num_rack_slots),
+        "vendor_allocation": [
+            {
+                "vendor": vendor,
+                "allocation_percent": vendor_allocation[vendor] / num_full_rack_slots,
+            }
+            for vendor in vendor_allocation
```

Comment:
SO pythonic of you

Reply:
thank you, pycharm encouraged me to do so.